web-ui/src/pages/PulsePage.jsx

```diff
@@ -102,6 +100,9 @@ const PulsePage = () => {
         data
       });
       if (res.error) throw new Error(res.error.message);
+
+      // Refresh browser to show that pulses where already submitted today.
+      location.reload();
```

Comment:
I noticed the page reloads nicely using `react-router-dom`'s `useHistory` hook like:

```js
import {useHistory} from 'react-router-dom';

const PulsePage = () => {
  const { state } = useContext(AppContext);
  const currentUser = selectCurrentUser(state);
  const csrf = selectCsrfToken(state);
  const history = useHistory();
```

Then the hook can be used to navigate the same page:

```js
history.push(location.pathname);
```

The user will then see a gentle page update and the thank you message.

Reply:
I made the change you suggested. Thanks for catching that!